{{cookiecutter.project_slug}}/MANIFEST.in

```diff
@@ -1,3 +1,4 @@
+graft {{ cookiecutter.project_slug }}
```

Comment:
Hey @luzfcb, thanks for the PR !
Have you checked if using this can conflict with the recursive-exclude below?
Namely, can it end up including `pyc` files or other normally ignored stuff?
Thanks!